drivers/SmartThings/zigbee-humidity-sensor/fingerprints.yml

```diff
@@ -1,4 +1,9 @@
 zigbeeManufacturer:
+  - id: "LUMI/lumi.sensor_ht.agl02"
+    deviceLabel: Aqara Temperature and Humidity Sensor T1
+    manufacturer: LUMI
+    model: lumi.sensor_ht.agl02
+    deviceProfileName: temp-humidity-battery
   - id: "eZEX/E282-KR0B0Z1-HA"
```

Comment:
Need to change the profile name. Please refer to other comment.

Reply:
The reason for creating a new Profile is to set the order of Temperature Capability first in order to display temperature instead of humidity in Card UI.

Reply:
seongyeol will send you a technical document how to reorder capability in ST app UI without define new profile.
Please update fingerprint once you got that document.
Thanks.

Reply:
@sbkimaqara 님, ( cc: @Kwang-Hui  )
I wrote guide for temp-humidity sensor.
Could you check this guide?
https://www.samsungmobilecoworking.com/its/browse/AQARAIOT-61

Reply:
Sure, I checked.